settings/src/components/account-status.js

```diff
@@ -36,6 +41,19 @@ export default function AccountStatus() {
 				${ backupCodesEnabled ? '' : 'not' }
 				verified your backup codes for two-factor authentication.`;
 
+	let svnBodyText = '';
+	if ( svnPasswordRequired || svnPasswordSet ) {
+		if ( svnPasswordRequired ) {
+			svnBodyText = svnPasswordSet
+				? 'You have an active SVN password set. You can request a new one at any time.'
+				: 'You have not set up an SVN password. This is required for SVN access to WordPress.org.';
+		} else {
+			svnBodyText = svnPasswordSet
+				? 'You have an active SVN password set. You can request a new one at any time.'
+				: 'You do not currently require WordPress.org SVN access.';
```

Comment:
"You not currently require.." seems like a slightly puzzsing inclusion. Should we just not show it at all? Or grey it out?

I assume it's here for users who don't have commit access to anything.

Reply:
There's 2 main states:
 1. The section is not shown
 2. The section is shown (Known core committer; plugin/theme author; **or has a SVN password set**; doesn't automatically trigger for things like.. buddypress commit)

The problem is that we don't actually know 100% when a user does or doesn't need a SVN password; we know probably 99.5% of the time.

Those then break down into:
 3. The user has a SVN password set (which is NOT the user_pass)
 4. The user doesn't yet have a SVN pass set.

Realistically; I think the text can be clarified significantly.